### DISCUSSION TITLE
Fat MachO binaries with x86_64h architecture type.

### DISCUSSION THREAD

**Discussion by torque59** [Vector35/binaryninja-api#1654] [General] [answered]
This might be an edge case, but fat MachO binaries with the above architecture type is not detected yet.

We might either want to redirect the user to use lipo, for the same as well or have a workaround for this if possible :).

**Comment by CouleeApps** [comment, marked as answer]
Looks like this is just missing a case in the Fat Mach-O loader. I've opened #1747 to track progress for this. A workaround for now is `lipo in_bin -extract x86_64h -output out_bin`.

**Reply by torque59** [reply]
Thanks @CouleeApps for tracking this , i was using lipo :)
